### ISSUE TITLE
Force refresh of harvest data?

### ISSUE THREAD

**Issue by newism** [mully/redmine_harvest#4]
Is it possible to force a refresh of all the harvest data for a particular project? Occasionally we make changes to Harvest comments (like adding task #ids) and they are not updated in Redmine.

**Comment by mully** [comment]
Not currently possible.  This would be a nice feature, though. 

Are you thinking about a "Sync All Harvest Data" button or something like that?

**Comment by newism** [comment]
Yeah exactly. It would be nice to do this on two levels, install wide and project wide.
